### PR TITLE
fix: cancelled booking doesn't show the booker

### LINKED ISSUE #8185: [CAL-1451] Cancelled booking doesn't show the booker any more
Someone booked a call with me. I cancelled it and now their details no longer show up on the booking when I view the booking link. It shows in the email correctly.

Email:

Booking:

[View the booking](https://app.cal.com/booking/cZkScdhAQTURUnHg9MA2EF?allRemainingBookings=false&email=)

<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-1451](https://linear.app/calcom/issue/CAL-1451/cancelled-booking-doesnt-show-the-booker-any-more)</sub>

### PR DESCRIPTION
**Pull request by G3root**
## What does this PR do?

Fixes #8185 

changes made:
- [x] prevent attendee deletion
- [x] added a e2e test  

## Type of change

- Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: packages/features/bookings/lib/handleCancelBooking.ts [hunk 1]
```diff
@@ -619,12 +619,6 @@ async function handler(req: CustomRequest) {
     return { message: "Booking successfully cancelled." };
   }
 
-  const attendeeDeletes = prisma.attendee.deleteMany({
```
Comment (hariombalhara) [inline packages/features/bookings/lib/handleCancelBooking.ts:622]: @zomars @emrysal Why were we deleting the attendees? There should be some reason for it.

**Review by hariombalhara — DISMISSED** [review]
Waiting on confirmation from @zomars @emrysal on preventing attendee deletion otherwise it's okay.
Blocking merge till then

**Comment by Udit-takkar** [comment]
@G3root any update on this PR ? is there anything else left

**Comment by G3root** [comment]
> @G3root any update on this PR ? is there anything left

I think this PR needs confirmation from @emrysal or @zomars

**Review by zomars — APPROVED** [review]
LGTM. Makes sense to keep record of the attendees. Unless the booking is completely deleted, we should keep them.

**Comment by zomars** [comment]
Let's fix the test first @G3root 
image

**Comment by G3root** [comment]
@alannnc any idea why's the test is failing ?

**Comment by alannnc** [comment]
> @alannnc any idea why's the test is failing ?
some data-testid changed but they weren't updated on the test files.

File: apps/web/pages/booking/[uid].tsx [hunk 2]
```diff
@@ -443,8 +443,10 @@ export default function Success(props: SuccessProps) {
                           )}
                           {bookingInfo?.attendees.map((attendee) => (
                             <div key={attendee.name + attendee.email} className="mb-3 last:mb-0">
-                              {attendee.name && <p>{attendee.name}</p>}
-                              <p data-testid={`attendee-${attendee.email}`}>{attendee.email}</p>
+                              {attendee.name && (
+                                <p data-testid={`attendee-name-${attendee.name}`}>{attendee.name}</p>
+                              )}
+                              <p data-testid={`attendee-email-${attendee.email}`}>{attendee.email}</p>
```
Comment (alannnc) [inline apps/web/pages/booking/[uid].tsx:449]: We needed to update test that were using attendee-${value}, so they keep working.
@G3root

**Pull request merged**